servers/lib/server/helpers.coffee

```diff
@@ -337,6 +337,112 @@ checkAuthorizationBearerHeader = (req) ->
 
   return token
 
+fetchGroupMembersAndInvitations = (client, data, callback, params) ->
```

Comment:
name `params` as callback, do not send the main callback to this function. calling code should handle the errors itself.